src/types/app.ts

```diff
@@ -54,4 +54,25 @@ export type Sponsor = {
   ];
 };
 
+export type Photo = {
+  data: [
```

Comment:
We dont need data.  Photo.data doesnt make sense.

Reply:
That's what we [received from CMS](https://cms.dltx.io/api/photos?populate=*). You did the same with Sponsors. 
No fix, then?

Reply:
Made changes